scripts/cibuild

```diff
@@ -54,7 +54,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
                   -f "${DIR}/../docker-compose.yml" \
                   -f "${DIR}/../docker-compose.airflow.yml" \
                   -f "${DIR}/../docker-compose.test.yml" \
-                  build airflow
+                  build airflow airflow-worker
```

Comment:
The airflow image doesn't have the rollbar token, so I can't use anything from the batch jar